src/lib/Components/Consignments/Screens/__tests__/Overview-tests.tsx

```diff
@@ -30,48 +32,50 @@ describe("Opening the right page", () => {
 
   it("pushes a Location when you tap on Locations", () => {
     overview.goToLocationTapped()
-    expect(navigator.push).toBeCalledWith({ component: Location, passProps: expect.anything() })
+    expect(navigator.push).toBeCalledWith({ component: Location, passProps: anything() })
   })
 
   it("pushes a Artist when you tap on Artist", () => {
     overview.goToArtistTapped()
-    expect(navigator.push).toBeCalledWith({ component: Artist, passProps: expect.anything() })
+    expect(navigator.push).toBeCalledWith({ component: Artist, passProps: anything() })
   })
 
   it("pushes a Metadata when you tap on Metadata", () => {
     overview.goToMetadataTapped()
-    expect(navigator.push).toBeCalledWith({ component: Metadata, passProps: expect.anything() })
+    expect(navigator.push).toBeCalledWith({ component: Metadata, passProps: anything() })
   })
 
   it("pushes a Provenance when you tap on Provenance", () => {
     overview.goToProvenanceTapped()
-    expect(navigator.push).toBeCalledWith({ component: Provenance, passProps: expect.anything() })
+    expect(navigator.push).toBeCalledWith({ component: Provenance, passProps: anything() })
   })
 
   it("pushes a SelectFromPhotoLibrary when you tap on Photos", () => {
     overview.goToPhotosTapped()
-    expect(navigator.push).toBeCalledWith({ component: SelectFromPhotoLibrary, passProps: expect.anything() })
+    expect(navigator.push).toBeCalledWith({ component: SelectFromPhotoLibrary, passProps: anything() })
   })
 })
 
 describe("Updating State", () => {
   let overview: Overview
   let stateMock: jest.Mock<any>
+  let update: any
 
   beforeEach(() => {
     overview = new Overview({ navigator, route, setup: {} })
+    update = overview.updateMetaphysics
     overview.setState = jest.fn()
     stateMock = overview.setState as any
   })
 
   it("updates Location", () => {
     overview.updateLocation("Huddersfield", "Yorkshire", "UK")
-    expect(stateMock).toBeCalledWith({ location: { city: "Huddersfield", country: "UK", state: "Yorkshire" } })
+    expect(stateMock).toBeCalledWith({ location: { city: "Huddersfield", country: "UK", state: "Yorkshire" } }, update)
   })
```

Comment:
This is the change from directly calling `setState` to moving to update metaphysics after.